packages/terra-modal-manager/examples/index-examples/ManagerContentComponent.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Button from 'terra-button';
+import ContentContainer from 'terra-content-container';
+import AppDelegate from 'terra-app-delegate';
+import ActionHeader from 'terra-clinical-action-header';
```

Comment:
With the new directory structure of examples being inside of the component package directory, you'll need to include components you use in your examples / tests as dependencies.

For example, the linter is complaining about:
```
7:26  error  Unable to resolve path to module 'terra-clinical-action-header'  import/no-unresolved
7:26  error  Missing file extension for "terra-clinical-action-header"        import/extensions
```

To resolve, add `terra-clinical-action-header` as dependencies to this component.

Reply:
Let's get this updated before we merge this. Talk to @mjhenkes or @emilyrohrbough if you need assistance resolving these lint errors.